app/experimenter/experiments/tasks.py

```diff
@@ -210,6 +210,23 @@ def send_period_ending_emails(experiment):
                 "Sent ending email for Experiment: {}".format(experiment)
             )
 
+    # send enrollment ending emails if enrollment end
+    # date is 5 days out
+    if experiment.enrollment_end_date:
+        if (experiment.enrollment_end_date - date.today()) <= timedelta(
+            days=5
+        ):
+            if not Experiment.objects.filter(
```

Comment:
Might be a dumb question... but is this supposed to be ExperimentEmail.objects.filter()?

Reply:
WOW.. oh my Gaia this was a great catch Tif!!! Can't believe I missed that and obviously I need to fix the tests as well, this should have been caught by testing!